### PR TITLE
importer: generate int data by histogram

### PR DESCRIPTION
**Pull request by hanfei1991**
pending to support more types.

### CODE REVIEW

File: cmd/importer/main.go [hunk 1]
```diff
@@ -51,9 +51,27 @@ func main() {
...
-		if err != nil {
-			log.Fatal(err)
+		statsInfo, err1 := loadStats(table.tblInfo, cfg.StatsCfg.Path)
+		if err1 != nil {
+			log.Fatal(err1)
+		}
+		for _, idxInfo := range table.tblInfo.Indices {
+			offset := idxInfo.Columns[0].Offset
+			if hist, ok := statsInfo.Indices[idxInfo.ID]; ok {
+				log.Warnf("index offset %d", offset)
```
Comment (coocood) [inline cmd/importer/main.go]: Why log warning?

File: cmd/importer/stats.go [hunk 2]
```diff
@@ -36,3 +39,45 @@ func loadStats(tblInfo *model.TableInfo, path string) (*stats.Table, error) {
...
 	return handle.LoadStatsFromJSON(tblInfo, jsTable)
 }
+
+type histogram struct {
+	stats.Histogram
+
+	index *model.IndexInfo
+}
+
+func (h *histogram) getRandomBoundIdx() int {
```
Comment (coocood) [inline cmd/importer/stats.go]: Please add comments about the difference between odd and even index.

**Comment by hanfei1991** [comment]
@coocood PTAL

**Comment by hanfei1991** [comment]
@WangXiangUSTC fixed by checking len of buckets after decoding stats file. PTAL

**Comment by WangXiangUSTC** [comment]
LGTM

**Pull request merged**